bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-options/container.jsx

```diff
@@ -1,48 +1,83 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 import logger from '/imports/startup/client/logger';
 import Auth from '/imports/ui/services/auth';
 import mapUser from '/imports/ui/services/user/mapUser';
 import Users from '/imports/api/users/';
+import Meetings from '/imports/api/meetings';
 import UserOptions from './component';
 
+
+const propTypes = {
+  users: PropTypes.arrayOf(Object).isRequired,
+  muteAllUsers: PropTypes.func.isRequired,
+  muteAllExceptPresenter: PropTypes.func.isRequired,
+  setEmojiStatus: PropTypes.func.isRequired,
+};
+
 export default class UserOptionsContainer extends Component {
   constructor(props) {
     super(props);
+    const meetingId = Auth.meetingID;
```

Comment:
``` const meetingId = Auth.meetingID;
 const meeting = Meetings.findOne({ meetingId });```
You have this in two spots. Instead you can bring it to the top as a single `const meeting = Meetings.findOne({ meetingId: Auth.meetingID });`